### ISSUE TITLE
gz-harmonic 1.0.0 fails on macos 14.2.1

### ISSUE THREAD

**Issue by davidbitton** [gazebosim/gz-harmonic#11]
## Environment
* OS Version: 14.2.1
* binary build : 1.0.0 (brew install)

## Description
* Expected behavior: successful launch of sim server
* Actual behavior: exception

## Steps to reproduce

1. `gz sim -v4 -r iris_runway.sdf -s`

## Output
```
E0000 00:00:1705795008.364901  801718 descriptor_database.cc:633] File already exists in database: gz/msgs/time.proto
```

**Comment by scpeters** [comment]
`iris_runway.sdf` is not provided by `gz-sim`; are you using [ardupilot_gazebo](https://github.com/ArduPilot/ardupilot_gazebo)?

**Comment by davidbitton** [comment]
yes. This error goes away when you remove the reference to the plugin. So, it's the plugin. I just can't find a place for a break point before it stops on that exception. I tried a source build (of gz), but i was unsuccessful; rpath issues.
